specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2020-06-01-preview/cosmos-db.json

```diff
@@ -4298,6 +4298,45 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.DocumentDB/locations/{location}/backupStorageRedundancies": {
+      "get": {
+        "operationId": "BackupStorageRedundancies_GetByLocation",
+        "x-ms-examples": {
+          "CosmosDBBackupStorageRedundanciesList": {
+            "$ref": "./examples/CosmosDBBackupStorageRedundanciesList.json"
+          }
+        },
+        "description": "Lists all the available backup redundancy types in a region.",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/locationParameter"
+          }
+        ],
+        "responses": {
+          "200": {
```

Comment:
A response to a collection GET needs to look like the paginated response body described here:  https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#get-resource

Reply:
Thanks for the doc, I will not put new API in this PR